src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/main.ts

```diff
@@ -29,6 +29,9 @@ const vuetify = createVuetify({
       mdi,
     },
   },
+  display: {
+    mobileBreakpoint: "sm",
```

Comment:
Have you sanity checked this throughout rest of application? What was this set to by default?

Reply:
It was set to Large by default. Which was strange. 

We're also just using this in the wizard right now. 

Other places we're using smAndUp so they aren't affected.